layers/place/city.sql

```diff
@@ -56,22 +56,22 @@ FROM (
                          row_number() OVER (
                              PARTITION BY LabelGrid(geometry, 128 * pixel_width)
                              ORDER BY "rank" ASC NULLS LAST,
-                                 place ASC NULLS LAST,
+                                 place ASC NULLS LAST, -- place here is type of place, one of 'city', 'town', 'village', 'hamlet', 'suburb', 'quarter', 'neighbourhood', 'isolated_dwelling'
                                  population DESC NULLS LAST,
                                  length(name) ASC
                              )::int AS gridrank
                   FROM osm_city_point
                   WHERE geometry && bbox
-                    AND ((zoom_level = 7 AND place <= 'town'::city_place
+                    AND ((zoom_level BETWEEN 4 AND 7 AND place <= 'town'::city_place
```

Comment:
Would you add a comment describing what/why for the zoom level change. I.e. 'adding additional intermediate results so that zooms 4 through 6 have additional city data available to use, so that ...'

Reply:
updated